package.json

```diff
@@ -4,15 +4,15 @@
   "description": "This is a project template for [Elder.js](https://elderguide.com/tech/elderjs/) apps. The template lives at https://github.com/elderjs/template and the Elder.js source is here: https://github.com/elderjs/elderjs",
   "main": "./build/build.js",
   "scripts": {
-    "start": "npm run dev",
-    "build": "run-s build:ts clean build:type-check build:html",
-    "build:type-check": "tsc -p tsconfig.json",
+    "start": "run-s dev",
+    "build": "run-s build:etsc build:ts clean build:html",
```

Comment:
@markjaquith So `build:type-check` doesn't do exactly the same as `build:ts`. Basically the typecheck uses the typescript compiler to validate the typescript where the other just builds.

Reply:
![CleanShot 2022-07-19 at 17 09 01@2x](https://user-images.githubusercontent.com/353790/179848950-29e7f847-7055-4689-8a07-5a5433111837.png)

They're literally the same command. Both do type checking, and neither one builds, because `tsconfig.json` has `"emit": false`.

Reply:
Ha. I should totally call it a day. 

Build ts should be `"build:ts": "etsc --config=etsc.config.cjs",`

Testing on multiple projects and bringing lessons back to the ts-template isn't working well. I'm losing track of where I am.

Reply:
Even better. Abstract away the "what is building TS" part.